lib/copyrightConstant.js

```diff
@@ -0,0 +1 @@
+module.exports = ['CC0', 'public', 'private']
```

Comment:
let this be an object instead of array.
copyright.cc0 copyright.public copyright.private